plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -782,7 +826,7 @@ public Iterator<TableColumnsMetadata> streamTableColumns(ConnectorSession sessio
                             return Stream.of();
                         }
                         String tableLocation = metastoreTable.get().location();
-                        TableSnapshot snapshot = transactionLogAccess.loadSnapshot(session, table, tableLocation);
+                        TableSnapshot snapshot = transactionLogAccess.loadSnapshot(session, table, tableLocation, Optional.empty());
```

Comment:
can we have a loadSnapshot overload with 3 arguments that call the 4argument method with a default Optional.empty()? it should make diff smaller

Reply:
```
public TableSnapshot loadSnapshot(ConnectorSession session, SchemaTableName table, String tableLocation)
            throws IOException
    {
        return loadSnapshot(session, table, tableLocation, Optional.empty());
    }
```